lib/download_test.go

```diff
@@ -21,14 +22,19 @@ func TestDownloadFromURL_FileNameMatch(t *testing.T) {
 	installPath := fmt.Sprintf(tempDir + string(os.PathSeparator) + ".terraform.versions_test")
 	macOS := "_darwin_amd64.zip"
 
-	// get current user
-	homedir, errCurr := homedir.Dir()
-	if errCurr != nil {
-		log.Fatal(errCurr)
+	home, err := homedir.Dir()
+	if err != nil {
+		log.Fatalf(`Could not detect home directory.`)
```

Comment:
Are backticks intentional (as opposite to double quotes)?

Reply:
No. I will fix that.